src/RamOperation.h

```diff
@@ -924,6 +965,48 @@ class RamIndexAggregate : public RamIndexOperation, public RamAbstractAggregate
     }
 };
 
+/**
+ * @class RamParallelIndexAggregate
+ * @brief Indexed aggregation on a relation
+ *
+ * TODO (rachel): Fix description!!
```

Comment:
This would be a good time to fix the description. If you want to expand on the brief version, then something about potentially using an index to find a smaller range for the aggregation? In parallel.